dotCMS/src/integration-test/java/com/dotcms/util/ImportUtilTest.java

```diff
@@ -622,100 +619,66 @@ private void validate(final Map<String, List<String>> results, final Boolean pre
+                        .contentTypeId(type.id()).dataType(
+                        DataTypes.TEXT).build();
+        titleField = fieldAPI.save(titleField, user);
+        fieldAPI.save(hostField, user);
+
+        workflowAPI.saveSchemesForStruct(new StructureTransformer(type).asStructure(),
+                Arrays.asList(schemeStepActionResult1.getScheme()));
 
-            assertTrue(results.get("warnings").size() == 1);
-            assertEquals("The Content Type field testTitle is unique.",results.get("warnings").get(0));
+        //Creating csv
+        reader = createTempFile("languageCode, countryCode, testTitle, testHost" + "\r\n" +
+                "es, ES, UniqueTitle, " + defaultSite.getIdentifier() + "\r\n" +
+                "en, US, UniqueTitle, " + defaultSite.getIdentifier() + "\r\n");
+        csvreader = new CsvReader(reader);
+        csvreader.setSafetySwitch(false);
+        csvHeaders = csvreader.getHeaders();
 
+        int languageCodeHeaderColumn = 0;
+        int countryCodeHeaderColumn = 1;
+        //Preview=false
+        results =
+                ImportUtil
+                        .importFile(0L, defaultSite.getInode(), type.inode(),
+                                new String[]{titleField.id()}, true, true,
+                                user, -1, csvHeaders, csvreader, languageCodeHeaderColumn,
+                                countryCodeHeaderColumn, reader,
+                                schemeStepActionResult1.getAction().getId());
+        //Validations
+        validate(results, true, false, true);
 
-        } finally {
-            try {
-                contentTypeApi.delete(type);
-            }catch (Exception e) { e.printStackTrace(); }
-        }
+        assertTrue(results.get("warnings").size() == 1);
+        assertEquals("The Content Type field testTitle is unique.", results.get("warnings").get(0));
     }
 
     @Test
     public void importFile_fails_when_twoLinesHaveSameUniqueKeys()
             throws DotSecurityException, DotDataException, IOException {
 
-        ContentType type;
         CsvReader csvreader;
         com.dotcms.contenttype.model.field.Field titleField, hostField;
         HashMap<String, List<String>> results;
         Reader reader;
         String[] csvHeaders;
-        long time;
-
-        //Creating new content type with one unique field
-        time = System.currentTimeMillis();
-
-        type = ContentTypeBuilder
-                .builder(BaseContentType.getContentTypeClass(BaseContentType.CONTENT.getType()))
-                .description("description" + time).folder(FolderAPI.SYSTEM_FOLDER)
-                .host(Host.SYSTEM_HOST)
-                .name("ContentTypeTestingWithFields" + time).owner("owner")
-                .variable("velocityVarNameTesting" + time)
-                .build();
 
-        type = contentTypeApi.save(type);
+        ContentType type = new ContentTypeDataGen().host(APILocator.systemHost()).nextPersisted();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'type' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4485019)